Models/Courses.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy.ext.declarative import declarative_base
+from  sqlalchemy import Column, Integer, String
+
+
+Base = declarative_base()
+
+
+class course(Base):
+    __tablename__ = 'courses'
+    department_ids = Column(String)
+    name = Column(String)
+    id = Column(Integer)
+    number = Column(Integer)
+
+    def __repr__(self):
+        return str(self.name)
```

Comment:
If you're looking for a unique hash, I'd just implement `__hash__` instead. IMO, `__repr__` is most used for debugging and logging (which is the difference between `__repr__` and `__str__` -- `__str__` is end-user facing, while `__repr__` is usually programmer / machine facing).